provider/sql.go

```
@@ -334,15 +340,16 @@ func (store *sqlOfflineStore) GetTransformationTable(id ResourceID) (Transformat
 	}
 	defer rows.Close()
 	if !rows.Next() {
-		return nil, fmt.Errorf("transformation not found: %v", name)
+		return nil, fferr.NewTransformationNotFoundError(name, id.Variant, err)
 	}
 	columnNames, err := store.query.getColumns(store.db, name)
```

error is never returned